src/config/pci-passthrough.md

```diff
@@ -0,0 +1,311 @@
+```
+
+```
+$ lspci -nnk -d 1002:aaf0 
+...
+        Kernel driver in use: vfio-pci 
+```
+
+The device ids we specified after the `vfio-pci.ids=` parameter should have the
+`vfio-pci` kernel driver in use.
+
+## Setting up OVMF-based guest VM
+
+Install the `qemu`, `libvirt`, and `virt-manager` packages. You must compile the
+`ovmf` package from source using xbps-src (see [building
+instructions](../packages/building.md)) after applying the patch located here:
+
+[New package: ovmf](https://github.com/void-linux/void-packages/pull/17225).
+
+Enable the following services with runit: `libvirtd`, `virtlogd`, and
+`virtlockd`.
+
+To run `virt-manager` without root, add yourself to the `libvirt` user group and
+uncomment the following lines in libvirt config `/etc/libvirt/libvirtd.conf`:
+
+```
+unix_sock_group = "libvirt"
+unix_sock_ro_perms = "0770"
+unix_sock_rw_perms = "0770"
+unix_sock_admin_perms = "0770"
+unix_sock_dir = "/var/run/libvirt"
+```
+
+Logout and log back in to ensure the new group settings are set up.
+
+Open `virt-manager` and create a new virtual machine. Go through all the steps
+as normal except at the end: check the "Customize before install" checkbox.
+
+- In "Overview" section set firmware to UEFI
+- In overview set chipset to Q35. See: ([Tips for PCI Passthrough -
+   momi.ca](https://momi.ca/2019/11/17/pcipassthrough.html#org44297d6))
+- In "CPUs" section, under Configuration, set "Copy host CPU configuration".
+
+Then finish your installation.
+
+Now that the install is done, add your PCI devices to your virtual machine. Open
+your virtual machine and click "Add Hardware" and switch to the "PCI Host
+Device". Add the devices we specified in the `vfio-pci.ids=` kernel parameter.
+
+Additionally, you may want to add a mouse and keyboard. You can do this by
+clicking "Add Hardware" and going to the USB Host Device section.
+
+> Note: To avoid having to use two mice and keyboard, I use a KVM switch and
+> passthrough one of my USB controllers. Your IOMMU groups may or may not allow
+> you to do this.
+
+Boot up your new virtual machine with your install medium and install your
+operating system.
+
+> Note: To install your guest operating system, make sure you plug a monitor
+> into the GPU you passed through.
+
+## Additional Tweaks
+
+> Note: You can edit your virt-manager config by running:
+> 
+> ```
+> # virsh edit <YOUR-VM-NAME>
+> ```
+
+It is highly recommended that you do the [performance
+tweaks](https://wiki.archlinux.org/index.php/PCI_passthrough_via_OVMF#Performance_tuning)
+from the ArchWiki.
+
+### Scream
+
+Audio on PCI passthrough are problematic to configure. One method to avoid this
+difficult configuration is to setup
+[scream](https://github.com/duncanthrax/scream). The required guest drivers can
+be found on github releases and the lines you need to add to your virt-manager
+config are
+[here](https://github.com/duncanthrax/scream#using-ivshmem-between-windows-guest-and-linux-host).
```

Comment:
"PCI passthrough audio can be difficult to configure. [Scream](https://github.com/duncanthrax/scream) can be used to avoid this. The required guest drivers can be found on the project's \[Releases page\](https://github.com/duncanthrax/scream/releases), and the relevant `virt-manager` configuration is described in \[the project's README\](https://github.com/duncanthrax/scream#using-ivshmem-between-windows-guest-and-linux-host)."